src/languages/en-US.ts

```diff
@@ -649,6 +649,7 @@ export default class extends Language {
 			'The starboard channel. If you star a message, it will be posted there. Using the `setStarboardEmoji` command allows the emoji customization.',
 		SETTINGS_STARBOARD_IGNORECHANNELS: 'The channels I should ignore when listening for new stars.',
 		SETTINGS_STARBOARD_MINIMUM: 'The minimum amount of stars required before a message is posted to the starboard channel.',
+		SETTINGS_STARBOARD_SELFSTAR: "If the author's reaction should be included in the total star count",
```

Comment:
```suggestion
		SETTINGS_STARBOARD_SELFSTAR: "Whether the author's reaction should be included in the total star count",
```

Consistency with other boolean keys (I think)